lib/commands/project/project.js

```diff
@@ -363,3 +385,110 @@ function spawn(command, options, cwd, cb) {
     cb(err);
   });
 }
+
+
+//.option('-p, --path-name [path]', 'a sepecific path of api')
+//.option('-f, --testmodule <module>', 'one of: ' + testmodules)
+//.option('-t, --assertion-format <type>', 'one of: ' + assertiontypes)
+//.option('-s, --asynchronous <boolen>', 'one of: ' + istrue)
+//.option('-o, --separate <boolean>', 'one of: ' + istrue)
+function testGenerate(options, cb) {
+  var template = require('swagger-test-templates');
+  var desiredPaths = [];
+  var testmodule = 'supertest';
+  var assertionFormat = 'expect';
+  var asynchronous = true;
+  var separate = false;
+
+  // parse swagger.yaml, get the json object
+  var yaml = require('js-yaml');
+  var string = fs.readFileSync('./api/swagger/swagger.yaml', {encoding: 'utf8'});
+  var project = JSON.parse(string);
+
+  // get the array of string paths from json object
+  var pathList = [];
+  var objectKeys = Object.keys(project.paths);
+  for (var i = 0; i < objectKeys.length; i++) {
+    pathList.push(objectKeys[i]);
+  }
+
+  //check if the test frame is one of the two
+  if (options.testmodule) {
+    testmodule = options.testmodule;
```

Comment:
`options.testModule = options.testModule || 'supertest'`
Hey @theganyo doesn't commander do the defaulting for us?